### DISCUSSION TITLE
作者大大，有一个小功能我想提一下意见，就是截图的时候，能不能增加一个设置

### DISCUSSION THREAD

**Discussion by ImnRa** [the1812/Bilibili-Evolved#5066] [提问 / Q&A]
因为浏览器有时候抽风或者自己手贱刷新的话，今天两次都是因为这两个原因导致一堆图都不见了，心态都炸了。所以，可以增加截一下就马上回车键下载一张图片的功能吗？或者是刷新后也能保留住之前截的图，除非自己点了全部丢弃

**Comment by the1812** [comment]
预览版已增加二次确认

**Reply by ImnRa** [reply]
我的天，居然更新了，谢谢大佬
